packages/suite/src/components/suite/modals/AddAccount/components/AddButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, Tooltip } from '@trezor/components';
+import { Translation } from '@suite-components';
+
+interface ButtonProps {
+    disabledMessage: React.ReactNode;
+    handleClick: () => void;
+}
+
+export const AddButton = ({ disabledMessage, handleClick }: ButtonProps) => (
+    <Tooltip maxWidth={285} content={disabledMessage}>
```

Comment:
I'd add a question icon tooltip when it's disabled or underline text at least, it's not clear there is an interaction (tooltip)

Reply:
yeah, would be nice to have some global solution for "disabled" buttons – I would imagine not to disable the buttons at all and display meaningful validation message when user click on button and some conditions to do the action are not met..

I don't think it makes sense to change it on on place in one way right now.